### PR TITLE
GNOM-2123 Using MD MiSeq at the Library Prep tab if you go back to Sa…

### PR DESCRIPTION
**Pull request by jdewell**
…mple Details the Sample Purification questions are displayed

### CODE REVIEW

**Comment by jdewell** [comment]
GNOM-2123 is the Jira issue I created for this bug.

**Comment by tcourdy** [comment]
Looks good to me

**Comment by matthewbjorge** [comment]
Looks all right to me as well

File: flex/views/experiment/TabSeqSetupView.mxml [hunk 1]
```diff
@@ -443,16 +443,16 @@
...
 					<mx:RadioButtonGroup id="seqPrepRadioGroup"/>
 					<mx:HBox id="hbox2" width="100%">
-						<mx:RadioButton id="samplesToBePreppedCheckbox"  value="Y" groupName="seqPrepRadioGroup"  label="Yes, please prep library" selected="true" click="{pickSeqPrepState()}">
+						<mx:RadioButton id="samplesToBePreppedCheckbox"  value="Y" groupName="seqPrepRadioGroup"  label="Yes, please prep library" selected="false" click="{pickSeqPrepState()}">
 						</mx:RadioButton>
 					</mx:HBox>
 
 					<mx:HBox >
-						<mx:RadioButton id="samplesAlreadyPreppedCheckbox"  value="N" groupName="seqPrepRadioGroup" label="No, library has already been prepped "
+						<mx:RadioButton id="samplesAlreadyPreppedCheckbox"  value="Y" groupName="seqPrepRadioGroup" label="No, library has already been prepped "
```
Comment (tcourdy) [inline flex/views/experiment/TabSeqSetupView.mxml]: Hey John -

Here you have set both of your radio group buttons for this group to a value of 'Y'.

Matt caught this and is going to check in the fix.

Thanks.

**Pull request merged**